docs/prerequisites/visual_studio.rst

```diff
@@ -20,6 +20,35 @@ For Visual Studio 2019
    page.
 -  Follow the steps prompted by the installer.
 
+After launching the Visual Studio installer:
+
+-  Select the desktop development with C++ workload and make sure it is
+   checked.
+-  Make sure the **Install while downloading** option is selected for
+   faster installation.
+-  Click on **Install**
+
+.. figure:: ../images/visual_studio/install_page.PNG
+   :alt: Install Section of Visual Studio 
+   :align: center
+
+   Install Section of Visual Studio 2019
+
+After the installation is complete:
+
+-  Restart your system.
+-  Create a folder and name it **Development**.
```

Comment:
Is there any preferred drive where this folder should be created? If yes, please add that.

Reply:
C:/ should be ideal as it is default for most configurations and used in the video.

Reply:
I don't think it makes a difference.